verl/trainer/config/sft_trainer.yaml

```diff
@@ -4,8 +4,12 @@ data:
   micro_batch_size_per_gpu: 4  # this is also val batch size
   train_files: ~/data/gsm8k/train.parquet
   val_files: ~/data/gsm8k/test.parquet
+  # Single-turn settings
   prompt_key: question
   response_key: answer
+  # Multi-turn settings
+  use_multiturn: false  # Set to true to use multi-turn dataset
```

Comment:
what about structure the config this way:
```
multiturn:
    enable: false
    messages_key: message
```
?

Reply:
Done!